translate/pb/pb_util.go

```diff
@@ -0,0 +1,103 @@
+package pb
+
+import (
+	"fmt"
+
+	"github.com/arr-ai/arrai/rel"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/reflect/protodesc"
+	pr "google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/reflect/protoregistry"
+	"google.golang.org/protobuf/types/descriptorpb"
+)
+
+// DecodeFileDescriptor parses protobuf definition file and decodes to file descriptor
+func DecodeFileDescriptor(definition []byte) (pr.FileDescriptor, error) {
```

Comment:
As breaking logic to 3 different functions in code `var stdProtobufDecode = rel.NewNativeFunction` of file `std_encoding.go`. And this code is in package syntax. 
It can change back and then will export `valueAsBytes` and `valueAsString` in package `syntax` as they are used in `var stdProtobufDecode = rel.NewNativeFunction`.